### PR TITLE
Contribute

### PR DESCRIPTION
**Pull request by saisankargochhayat**
## Related Issues and Dependencies
Added contribute.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+To make sure our code is [PEP8](https://www.python.org/dev/peps/pep-0008/) compliant, we use:
+
+- [black code formatter](https://github.com/psf/black)
+- [Flake8 code linter](http://flake8.pycqa.org)
+- [mypy static type checker](http://mypy-lang.org)
+
+There's a [pre-commit](https://pre-commit.com) config file in [.pre-commit-config.yaml](.pre-commit-config.yaml).
+To [utilize pre-commit](https://pre-commit.com/#usage), install pre-commit with `pip3 install pre-commit` and then either:
+
+- `pre-commit install` - to install pre-commit into your [git hooks](https://githooks.com). pre-commit will from now on run all the checkers/linters/formatters on every commit. If you later want to commit without running it, just run `git commit` with `-n/--no-verify`.
```
Comment (harshad16) [inline CONTRIBUTING.md:66]:
```suggestion
- `pre-commit install` - Install pre-commit into your [git hooks](https://githooks.com). 
pre-commit will run all the checkers/linters/formatters for every commit. 
If you want to commit without running it, just run `git commit` with `-n/--no-verify`.
```
how about something along this line.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+- Pipfile
+- setup.py if its a package. 
+
+### Documentation
+
+If you want to update documentation, [README.md](README.md) is the file you're looking for.
+
+
+### How to contribute code to ogr
```
[the remark this answers is not in the document]
Comment (saisankargochhayat) [reply CONTRIBUTING.md]: Thanks changed it.

**Comment by goern** [comment]
Uhhh, noice!! Can you blend in https://github.com/thoth-station/core/pull/234

**Comment by saisankargochhayat** [comment]
retest

**Comment by saisankargochhayat** [comment]
recheck

**Pull request merged**